EXAM 2 REVIEW
Know the following:
## Use of Edman degradation to remove one amino acid at a time from N-terminus of peptide or protein (you don’t need to know the reaction steps)
## Enzymatic and chemical ways to cleave a protein into oligopeptides for amino acid sequencing
    ### Aspartic Proteases
        - Twp asp at the active site
        - Use Low barrier hydrogen bond
        - Example is HIV Protease(Homo Diamer)
    ### Serine Proteases
        + All three have three polar residues
            * Ser
            * His
            * Asp
        + Mechanism
            - Asp forms LBHB with Substrate
            - Histodine acts as the general acid and base
            - Serine creates a covalent bond with the peptide that is going to be cleaved
            - That bond changes trigonal C to tetrahedral C
            - Two negative tetrahedral anions are formed
        - Chymotrypsin
            * Cleaves Phe, Trp, Tyr
            * Cleaves Aromatics except when followed by proline
            * Cleaves on Carbonyl Side
            + Substrate Binding Pocket
                - Hydrophobic
        - Trypsin
            * Cleaves Arg and Lys
            * Except when followed by __proline__
            * Cleaves Carbonyl side
            + Substrate Binding Pocket
                - Negatively charged
        - Elastase
            * Cleaves small amino acids, that are neutral
            * Cleaves on Carbonyl
                - Except when followed by Proline
            + Substrate Binding Pocket
                - Bulky Residue at the top
## Mass spectrometry: what MADLI and ESI are and how mass spectrometry is used to separate and sequence peptides
### ESI
+ Connected to Chromotagraphy System
+ Set at pH 2
+ Everything is protanated
+ Charges   
    - +1 for N terminus
    - +1 for Arg, His, Lys
+ Each one to the left has one more charge than the next
### MADLI
+ Adds a single proton to the WHOLE protein
+ Uses laser matrix to move the proton
+ Peaks are individual proteins
### How do you calculate mass?
(M+Z)/Z
## What BLAST is and how it is used to detect protein homology; typical conservative amino acid replacements during protein evolution
+ Basic Local Alignment Tool
Used to check how similar two sequences are.
Gives:
+ Orthologs \- Similar Sequence __Different__ Species
+ Paralog \- Similar Sequence in __Same__ Species
## Average packing density of 0.74 for atoms in protein
# COME BACK TO Definitions for levels of protein structure, including supersecondary and domains
## Planar structure and cis/trans configuration of peptide bond unit; reason for and consequences on protein structure
Peptide bonds __RESEMBLE__ double bonds because:  
    + Shorter than single bond
    + __BUT__ longer than double bond
    + Dipole
        - N partially +
        - O partially -
    + Most commonly in Trans because of Steric interference
Significance of this is that the Peptide bond cannot __ROTATE__ making the C,N,O __Coplaner__
## Definition of φ and ψ angles; φ angle of pro is fixed
Memorize Picture
## What a Ramachandran plot shows; why glycine can be located anywhere on the plot
+ Alpha Helix - Top of BOTTOM Left
+ Beta Sheet - Top Left
#### Glycine
Can be found because it has __no__ β-carbon
# COME BACK TO
Types of secondary structure: α, β, etc; their characteristics, e.g., pitch, handedness, foldedness and placement of side chains
Why α helix and β conformation predominate (i.e., because they represent minimum free energy
states)
Why parallel β sheets are in the interior of proteins and are less stable than antiparallel sheets (viz., less favorable skewed H-bonds can’t compete with H-bonding to water)
To recognize the pattern of hydrophobic residues alternating with hydrophilic residues in polypeptide strands in antiparallel sheets at the surface of proteins

## To recognize the pattern of Gly-X-Pro or Gly-Pro-X found in collagen-like strands
Five types of supersecondary structure, including α helix coiled-coil (e.g. leucine zipper with heptad repeats having leu or similar amino acid at positions 1 and 4 of 7-residue repeats)
Structures of β barrels, β sandwiches, and β propellers as bases of domains; what supersecondary structures they contain
Properties of IUPs (Intrinsically Unstructured Proteins)
## Names of the six E.C. classes of enzymes and be able to recognize their reactions
### Oxidoreductase
NAD --> NADH  
Reduced
### Transferase
Transfers something on the substrate
Uses ATP Phosphorolation
Transfers __Pi__ to the substrate
### Hydrolase
Adds water
### Lyase
Breaks apart
### Isomerase
Cis to Trans
Trans to Cis
### Ligase
Uses __ATP__ energy for Ligation
Binds together
## RNA enzymes are called ribozymes, which are limited in reactions catalyzed due to presence of only –OH catalytic groups on the ribose
### The peptidyl transferase of ribosomes is a ribozyme
## Difference between Michaelis and allosteric enzymes
### Michaelis-Menton
+ The ones we use the graphs on
+ Rapid equalibrium
+ __K-1__ >> __K2__
+ Steady State assumption
    - Assume substrate is in excess
    - Assume substrate is in steady state
    - __Km__ = __Ks__ when __K-1__ >> __K2__
### Allosteric
+ More than __two__ subunits
+ AND more than __two__ active sites
## Michaelis-Menton and Briggs-Haldane equations; be able to use to solve problems
See Equation Sheet
## Understand meaning of K S , K m , k cat (turnover number), and k cat /K m ratio
### KS
Dissociation Constant of the Substrate
### Km
1/2 Vmax
### Kcat
Number of __Substrate molecules__ converted to __product__ per __enzyme__ per __time__
### Kcat / Km ratio
Catalytic Efficiency
E = Kcat/Km
## Know expected cellular [S] based on K m and efficiency of perfect enzyme based on k cat /K m ratio
Kinetic basis for enzyme assay
How K m is determined
The three types of reversible enzyme inhibitors; their distinguishing effects on K m and V max ;
and their diagnostic Lineweaver-Burk plots
Formulas for K m apparent and V max apparent for different types of inhibition, e.g. K m app =
K m (1+[I]/K I ) for competitive inhibition; be able to solve for K I or true K m
How sulfa drugs and viagra work and why ethanol is an antidote for methanol or anti-freeze
poisoning
Types of bisubstrate reactions and their diagnostic Lineweaver-Burk plots (Don’t need to know
the equations)
Other ways to diagnose type of bisubstrate reaction (i.e., ligand binding studies and exchangereactions)
What a Scatchard plot shows
How irreversible inhibitors work (they covalently bond to enzymes) and several examples:
organophosphorus nerve gases and insecticides, penicillin (suicide substrates)
Enzymes catalyze reactions by lowering activation energy, not by changing ΔG for the reaction
The seven reasons listed in lecture notes for catalysis by enzymes
The properties of NACs (Near-Attack Conformations)
The amino acids that can form unstable covalent intermediates during catalysis (ser, thr, tyr,
cys, lys, his, asp, glu) and the types of unstable covalent bonds formed, including Schiff base
formation with lys
Characteristics of general acid and general base catalysis and the amino acids that can act as
general acid and general base catalysts (asp, glu, his, cys, tyr, lys)
Ways pKa values can be shifted to permit catalysis (i.e. exam 1 info)
Properties of LBHBs (Low-Barrier H-Bonds) and role in catalysis
Characteristics of metal ion catalysis with Zn 2+ as an example
The catalytic amino acids and their roles in the serine (also applies to cysteine) and asp
proteases
Types and examples of protein denaturants and how they work
T m of a protein is the melting temperature, where 50% of the protein molecules are denatured
Chaperones: catalysts for protein folding – Hsp70 and chaperonin (aka. Hsp60) – and how they work using ATP hydrolysis to drive conformational changes

